app/src/main/java/com/example/trempelapp/data_layer/inMemory/SharedPreferencesManagerImpl.kt

```diff
@@ -18,8 +17,15 @@ class SharedPreferencesManagerImpl @Inject constructor(
         context.getSharedPreferences(TOKEN_PREFERENCES_FILE, Context.MODE_PRIVATE)
 
     override fun getUserLoginStatus(): Single<String> {
+        writeToken("")
```

Comment:
but you will rewrite token every time when you are trying to get token. It's incorrect

Reply:
it was done to check if the app was working correctly

Reply:
done